docs-android/app/src/main/res/values-de/strings.xml

```diff
@@ -9,6 +9,7 @@
     <string name="validate_error_alphanumeric">Nur Buchstaben und Zahlen</string>
 
     <!-- App -->
+	<string name="app_name" translatable="false">Sismics Docs</string>
```

Comment:
```suggestion
```
If you see translatable="false", don't put it in the German file, it doesn't need to be translated.